app/views/admin/booths/_form.html.haml

```diff
@@ -0,0 +1,33 @@
+.row
+  .col-md-12
+    .page-header
+      %title Request a Booth
+.row
+  .col-md-8
+    = semantic_form_for(@booth,  url: @booth.new_record? ?  admin_conference_booths_path(@conference.short_title) : admin_conference_booth_path(@conference.short_title, @booth.id), html: { multipart: true }) do |f|
+      = f.input :title, as: :string, required: true
```

Comment:
When the form page loads, I expect to be typing in this field, so this field should have autofocus, then the user doesn't have to click in it.